src/components/grid/grid.spec.js

```diff
@@ -148,8 +148,15 @@ describe('Grid', () => {
     /* the aim here is not to test that CSS media queries work. we are simply */
     /* checking that styled components are built and applied correctly        */
     it('builds the style rules for the GridItem responsiveSettings', () => {
-      const elem = mount();
-      expect(elem).toMatchSnapshot();
+      expect(
+        shallow(
+          <GridContainer id='testContainer'>
+            <GridItem responsiveSettings={ [item11500, item11300, item1900] }>1</GridItem>
+            <GridItem responsiveSettings={ [item21500, item21300, item2900] }>2</GridItem>
+            <GridItem responsiveSettings={ [item31500, item3900, item31300] }>3</GridItem>
+          </GridContainer>
+        )
+      ).toMatchSnapshot();
```

Comment:
I think that the main reason of this snapshot was to render CSS styles, which are missing on shallow render

Reply:
Yeah fair point, I've updated to use `TestRenderer.create` as it will maintain the css